src/sentry/integrations/utils/code_mapping.py

```diff
@@ -61,10 +67,8 @@ def stacktrace_buckets(self, stacktraces: List[str]) -> Dict[str, List[FrameFile
                     buckets[bucket_key] = []
                 buckets[bucket_key].append(frame_filename)
 
-            except ValueError:
```

Comment:
I have not been able to determine what I was testing for in here or what test to write to trigger it, thus, making the exception a bit more generic.

This is the only block of the file that has no code coverage.